src/Lean/Server/References.lean

```diff
@@ -117,8 +134,8 @@ open Elab
 
 /-- Content of individual `.ilean` files -/
 structure Ilean where
-  version : Nat := 1
-  module : Name
+  version    : Nat := 2
```

Comment:
Why is there a version bump if the format hasn't changed? Is the semantics different?

Reply:
The format has changed because `RefInfo` changed to store the parent declaration. See the PR description.